hijri/07-rajab.yml

```diff
@@ -132,7 +132,7 @@ events:
       - شیعه
   sources:
     - https://fa.wikipedia.org/wiki/موسی_کاظم#حبس_و_کشته‌شدن
-- partial_key: "muhammad_first_revelation"
+- partial_key: "ascension_of_prophet_mohammad"
```

Comment:
This `muhammad_first_revelation` is the [english wiki page of this event](https://en.wikipedia.org/wiki/Muhammad%27s_first_revelation). it is better to keep it as is.

Reply:
I saw the Wiki link and I think both are the same but the new one has a direct reference to the event. anyways your comment is right if we should focus on the Wiki page.